src/annotator/components/ContentInfoBanner.js

```diff
@@ -23,13 +24,18 @@ import {
  *   @param {ContentInfoConfig} props.info
  */
 export default function ContentInfoBanner({ info }) {
+  // Format item title to show subtitle
+  let itemTitle = info.item.title;
+  if (info.item.subtitle) {
+    itemTitle += `: ${info.item.subtitle}`;
+  }
```

Comment:
So: Showing subtitle for the item itself, but not the container, because we lack the real estate.